CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## 0.8.0-dev
 
+### Added
+
+- Support for colored underlines
```

Comment:
I think this entry isn't entirely clear. You can already have colored underlines in Alacritty, you just can't change it from the text color.

Maybe something like this is a little less confusing:


```
- Underline color escape (`CSI 58 : 2 : Ps : Ps : Ps m`/`CSI 58 : 5 : Ps m`)
```